tests/test_wheel.py

```diff
@@ -54,6 +54,10 @@
     SHARED_LIB_EXT = 'dylib'
 elif platform.system() == 'Windows':
     SHARED_LIB_EXT = 'pyd'
+elif platform.system().startswith('CYGWIN'):
+    # I'm using import library (link-time) rather than dynamic library (run-time)
+    # because that's what was already here for Cygwin
+    SHARED_LIB_EXT = 'dll.a'
```

Comment:
This should be `.pyd` as well I guess? Either that or `.so` or `.dll`. But `dll.a` is the import library, not the extension module.